db/migrate/20210526091948_add_column_to_hall.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToHall < ActiveRecord::Migration[6.1]
+  def change
+    add_column :halls, :hall_number, :integer
```

Comment:
you have column `hall_number` but in API you use it as a name 
it's not a big problem sometimes you have one column but in few cases you call them with another name
but what if someone wants to name hall as "big hall" or "main hall" or "whatever" and now you can only name it as integer